### PR TITLE
Add a disclaimer and acceptance of disclaimer before bucket-wipe is e…

### PR DESCRIPTION
**Pull request by xiaoxin-ren**
…xecuted.

### CODE REVIEW

File: src/main/java/com/emc/ecs/tool/BucketWipe.java [hunk 1]
```diff
@@ -126,6 +142,9 @@ private static Options options() {
                 .desc("instead of listing bucket, delete objects matched in source file key list").build());
         options.addOption(Option.builder().longOpt("delete-mpus").desc("incomplete MPUs will prevent the " +
                 "bucket from being deleted. use this option to clean up all incomplete MPUs").build());
+        options.addOption(Option.builder("y").longOpt("accept-disclaimer").argName("accept-disclaimer")
```
Comment (twincitiesguy) [inline src/main/java/com/emc/ecs/tool/BucketWipe.java]: this should not have an argument (no `argName`)

**Review by twincitiesguy — CHANGES_REQUESTED** [review]
Need unit tests for positive and negative cases with the new option

File: src/main/java/com/emc/ecs/tool/BucketWipe.java (hunk 1 above)
Comment (xiaoxin-ren) [reply src/main/java/com/emc/ecs/tool/BucketWipe.java]: fixed

**Review by twincitiesguy — APPROVED** [review]
